vmdb/app/models/ems_refresh/parsers/foreman.rb

```diff
@@ -93,7 +93,7 @@ def configured_system_inv_to_hashes(recs, profiles, operatingsystems)
             :configuration_profile      => id_lookup(profiles, cs, "hostgroup_id"),
             :operating_system_flavor_id => id_lookup(operatingsystems, cs, "operatingsystem_id"),
             :last_checkin               => cs["last_compile"],
-            :build_state                => cs["build"] == "true" ? "pending" : nil,
+            :build_state                => cs["build"] ? "pending" : nil,
```

Comment:
Is cs["build"] a boolean or a String?  If it's a string, this will always end up being "pending"